eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -33,79 +33,86 @@ parameters:
     type: string
     default: ''
 steps:
-- template: /eng/common/pipelines/templates/steps/enable-long-path-support.yml
+  - ${{ if eq(length(parameters.PackageInfoLocations), 0) }}:
```

Comment:
Why does this PR have eng/common changes in it? Those files shouldn't be manually modified/merged per repo since they are shared across languages.
https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/common_engsys.md#updating

Are these just fixes for a previous commit/merge that got us out of sync?

Reply:
Oh, I see, this PR includes a bunch of changes to sync between `main` and the `feature` branch. That makes sense.